docs/readme.md

```diff
@@ -0,0 +1,77 @@
+# Pym Shortcode
+
+Pym Shortcode will resize an iframe responsively depending on the height of its content and the width of its container. The plugin uses [Pym.js](http://blog.apps.npr.org/pym.js/), developed by the NPR Visuals Team, to allow embedded content in WordPress posts and pages using a simple shortcode. Like Pym.js it also bypasses the usual cross-domain issues.
+
+## Installation
+
+1. Download the zip file from this Github repo and upload it manually to your WordPress site.
```

Comment:
same deal here, need instructions for adding it this way + from the wp.org directory